compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs

```diff
@@ -31,6 +31,10 @@ version1_7 = V1 $ PointStable 7
 version1_8 :: Version
 version1_8 = V1 $ PointStable 8
 
+-- | DAML-LF version 1.11
+version1_11 :: Version
+version1_11 = V1 $ PointStable 11
```

Comment:
Just a remark (I do no request any change here):

1.11 is not recognized as "stable" by the engine, meaning ledger should be started with '--dev-mode-unsafe'.
In Scala land, we dropped the Variant to distinguish between "stable" and "dev" versions, we use instead the Engine `Config` to forbid loading LF > 1.8 when started in "stable" mode.

Reply:
I don't think we need to change anything here since the notion of stable/dev is completely an internal thing in `damlc`. We could introduce something like `PointPreview` but I'm not sure how much that buys us.